Makefile

```diff
@@ -1,15 +1,36 @@
-QA_DOCKER_IMAGE=jakzal/phpqa:alpine
+QA_DOCKER_IMAGE=rollerworks/search-phpqa:latest
 QA_DOCKER_COMMAND=docker run -it --rm -v "$(shell pwd):/project" -w /project ${QA_DOCKER_IMAGE}
-ELASTICSEARCH_DOCKER_COMPOSE=lib/Elasticsearch/docker-compose.yml
 
-dist: cs-full phpstan test-full
+dist: install cs-full phpstan test-full
 ci: cs-full-check phpstan test-full
 
+install:
+	composer.phar install --no-progress --no-interaction --no-suggest --optimize-autoloader --ansi
+
+install-dev:
+	cp composer.json _composer.json
+	composer.phar config minimum-stability dev
+	composer.phar update --no-progress --no-interaction --no-suggest --optimize-autoloader --ansi
+	mv _composer.json composer.json
+
+install-lowest:
+	composer update --no-progress --no-suggest --prefer-stable --prefer-lowest --optimize-autoloader --ansi
+
 test:
 	vendor/bin/phpunit --verbose --exclude-group functional,performance
 
-test-full: es-up
+test-full:
+	export SYMFONY_DEPRECATIONS_HELPER=strict
+
+	# These tasks should be executed in paralel.
```

Comment:
https://www.gnu.org/software/parallel/

Reply:
Yes, I know. I already tried this in the past with the script symfony/symfony uses but failed to make it work then 😃

Reply:
Just tried it, don't think you actually need to mess with parallel here, DB specific tests here are 7sec on my system in total while the regular part is ~30s.